package/hoff.service

```diff
@@ -4,6 +4,12 @@ After=network.target
 Requires=network.target
 AssertPathExists=/etc/hoff/config.json
 
+# Generic crash reporting. This makes systemd start the on-failure@hoff.service
+# unit when the process exits or fails to start. Comment this out if you do not
+# have a systemd unit with this name. See `package/on-failure@.service.example`
+# for an idea of what you can do with this.
+OnFailure=on-failure@%n
```

Comment:
What happens when `on-failure@hoff.service` does not exist?

Reply:
The service will start fine. When it crashes, there will be a log message that the OnFailure handler wasn't found. Then systemd will attempt to restart the service.

Tested with:

```
# /etc/systemd/system/foo.service
[Unit]
Description=Test
OnFailure=doesnotexist@%n

[Service]
Type=simple
ExecStart=/bin/false
Restart=on-failure
RestartSec=10
```